docker/boost-client/entrypoint.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
```

Comment:
We can just still add it but only put a boost -v command so that the user has confirmation that boost is installed.

```
#!/usr/bin/env bash
set -e
boost -v
```